plugins/base/src/test/resources/expect/signatureTest/out/styles/style.css

```diff
@@ -0,0 +1,353 @@
+@import url(https://fonts.googleapis.com/css?family=Open+Sans:300i,400,700);
```

Comment:
Are you sure this is signature specific file? If it does not, it should definitely be moved out of this PR

Reply:
This files are compared to output generated during tests, so if I'd remove certain parts tests would fail.